### PR TITLE
Fix the problem that backup time may be negative

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve? 
#396 made a little mistake, which make `last backup finish time - now` become backup time spent, and will lead to backup time become negative.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
image

### Release Note

 - No release note

### CODE REVIEW

File: pkg/backup/client.go [hunk 1]
```diff
@@ -405,13 +405,14 @@ func (bc *Client) BackupRanges(
...
+		lastBackupStart, currentBackupStart := init, init
 		for files := range filesCh {
-			cur, start = start, time.Now()
+			lastBackupStart, currentBackupStart = currentBackupStart, time.Now()
 			allFiles = append(allFiles, files...)
-			summary.CollectSuccessUnit("backup ranges", 1, cur.Sub(start))
+			summary.CollectSuccessUnit("backup ranges", 1, currentBackupStart.Sub(lastBackupStart))
 		}
-		log.Info("Backup Ranges", zap.Duration("take", cur.Sub(init)))
+		_ = lastBackupStart
```
Comment (3pointer) [inline pkg/backup/client.go:414]: 🤔️

**Comment by YuJuncen** [comment]
@kennytm if we remove that line, golang-ci will get unhappy:

```
pkg/backup/client.go:408:23: ineffectual assignment to `lastBackupStart` (ineffassign)
 		currentBackupStart, lastBackupStart := init, init
```

It's really strange cuz if we leave `currentBackupStart` unused everything is alright but `lastBackupStart` cannot.
Maybe it wants me to move it into the inner scope, but I think the current version is more readable, and add this line to make CI happy.

File: pkg/backup/client.go (hunk 1 above)
Comment (YuJuncen) [reply pkg/backup/client.go]: OK. Let me replace it with `// nolint`...

**Comment by kennytm** [comment]
@YuJuncen i think it's because `lastBackupStart` is immediately rewritten inside the for loop and unused outside the for loop. Maybe you could make it `var lastBackupStart time.Time` to silence the warning.

**Comment by YuJuncen** [comment]
@kennytm I think so, but I used `// nolint:ineffassign` to silence it.

**Pull request merged**